gio/src/file.rs

```diff
@@ -1,13 +1,13 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
-use std::{cell::RefCell, ffi::CStr, mem, pin::Pin, ptr};
+use std::{cell::RefCell, mem, pin::Pin, ptr};
```

Comment:
Can you add a closes  https://github.com/gtk-rs/gtk-rs-core/issues/40 to this commit?